compiler/noirc_evaluator/src/ssa.rs

```diff
@@ -144,17 +148,27 @@ impl SsaProgramArtifact {
     }
 }
 
+pub struct SsaEvaluatorOptions {
+    /// Emit debug information for the intermediate SSA IR
+    pub enable_ssa_logging: bool,
+
+    pub enable_brillig_logging: bool,
+
+    /// Force Brillig output (for step debugging)
+    pub force_brillig_output: bool,
+
+    /// Pretty print benchmark times of each code generation pass
+    pub print_codegen_timings: bool,
```

Comment:
💭 I'm not sure which final names should we use here ("enable_ssa_logging" vs. "show_ssa" vs. "print_ssa_passes"). Maybe using all names would be better, but then using different names at different layers might also make sense.